articles/custom-domains/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Custom Domains
+description: How to map custom domains
+toc: true
+---
+# Custom Domains
+
+Auth0 allows you to map your the domain for your tenant to a custom domain of your choosing. This allows you maintain a consistent experience for your users by keeping them on your domain instead of redirecting or using Auth0's domain. For example, if your Auth0 domain is `northwind.auth0.com`, you can have your users to see, use, and remain on `login.northwind.com`.
+
+## Prerequisites
+
+You'll need to register and own the domain name to which you're mapping your Auth0 domain.
+
+## Features Supporting Use of Custom Domains
+
+Currently, the following Auth0 features and flows support use of custom domains:
+
+* OAuth 2.0/OIDC-Compliant Flows (all of which make calls to the [`/authorize` endpoint](/api/authentication#authorize-client))
+* Guardian (Version 1.3.3 or later)
+* Emails -- the links included in the emails will use your custom domain
+
+## Certificates
+
+You can manage the certificates for your custom domains, or you can have Auth0 manage them on your behalf.
+
+If you opt to manage your own certificates, you'll need to use a reverse proxy that you configure using AWS CloudFront or Microsoft Azure.
+
+### Auth0-Managed Certificates
+
+::: warning
+Auth0-Managed Certificates is a beta feature.
+:::
+
+Auth0 will manage the creation and renewal of the certificates for your custom domain. This is the simplest custom domains deployment option.
+
+### Self-Managed Certificates
+
+You are responsible for managing your SSL/TLS certificates and configuring a reverse proxy to handle SSL termination and the forwarding of requests to Auth0. Choose this option if you wish to have more control of your certificates (such as choosing your own CA or certificate expiration), or if you want to have more monitoring over your API calls to Auth0.
+
+## How to Configure Custom Domains
+
+Setting up your custom domain requires you to do the following steps:
```

Comment:
Here the steps will differ if you are using Self-Managed Certs or Auth0-Managed Certs.

The steps you have here are fine for Self-Managed, but for Auth0-Managed you must:
- Create the custom domain
- In dashboard, you will be instructed to create the cname record in DNS
- Come back when that is done to verify (note it may take a few minutes or more to verify)
- It's ready.